client/my-sites/site-settings/section-import.jsx

```diff
@@ -16,31 +18,26 @@ import MediumImporter from 'my-sites/importer/importer-medium';
 import { fetchState } from 'lib/importer/actions';
 import { appStates, WORDPRESS, MEDIUM } from 'state/imports/constants';
 import EmailVerificationGate from 'components/email-verification/email-verification-gate';
+import { getSelectedSite } from 'state/ui/selectors';
 
-export default React.createClass( {
-	displayName: 'SiteSettingsImport',
+class SiteSettingsImport extends Component {
+	static propTypes = {
+		site: PropTypes.object,
```

Comment:
`propTypes.site` is less descriptive than previously. It would be nice to maintain the `shape` with an additional `ID` property.